code/parachain/frame/composable-support/src/collections/vec/bounded/bi_bounded_vec.rs

```diff
@@ -55,6 +55,16 @@ pub enum BiBoundedVecOutOfBounds {
 	},
 }
 
+impl<T, const U: usize> BiBoundedVec<T, 1, U> {
```

Comment:
this guy needs > 1. and it gives a lot. by author does no evolves it (  . until comparisons this approach is not yet great.

Reply:
with const vector